packages/vitest/src/node/stdin.ts

```diff
@@ -1,6 +1,9 @@
 import readline from 'node:readline'
 import c from 'picocolors'
 import prompt from 'prompts'
+import ansiEscapes from 'ansi-escapes'
+import { relative } from 'pathe'
+import type { Task, Test } from '@vitest/runner/types/tasks'
```

Comment:
`/types/tasks` is not exposed so it will fail at runtime, only `/types` is

Reply:
Thank you, I have removed them.